.github/workflows/build.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       - name: Format
         run: |
           cargo fmt --all -- --check
-          cargo clippy
+          cargo clippy --workspace --tests --bins -- -D warnings
```

Comment:
Should we really include tests here? Just thinking that often tests are better when they violate some of the recommendations. E.g. we often don't care about unwraps in tests, since it's less code to read and supposed to function only in one specific scenario.

Reply:
Yes
Clippy recommendations aren't crucial in tests.